src/app/containers/ChartbeatAnalytics/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react';
+import useToggle from '../Toggle/useToggle';
+import AmpChartbeatsBeacon from './amp';
+import CanonicalChartbeatsBeacon from './canonical';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import { RequestContext } from '../../contexts/RequestContext';
+import {
+  chartbeatUID,
+  useCanonical,
+  getSylphidCookie,
+  getDomain,
+  buildSections,
+  getType,
+} from '../../lib/analyticsUtils/chartbeat';
+
+const ChartbeatAnalytics = () => {
+  const { service } = useContext(ServiceContext);
+  const { env, platform, pageType } = useContext(RequestContext);
+  const { enabled } = useToggle('chartbeatAnalytics');
+
+  if (enabled) {
```

Comment:
Can we fail early instead so we don't have to wrap all this logic inside a conditional?

```js
if (!enabled) {
    return null;
}
```